database/password.js

```diff
@@ -0,0 +1,3 @@
+const PASSWORD = 'FILL_ME_IN';
+
+module.exports.PASSWORD = PASSWORD;
```

Comment:
Is this a file you would want added to your .gitignore as well?

Reply:
Nope. This one is supposed to be here so that people know how my password that is ignored looks.